test/prototype/test_awq.py

```diff
@@ -70,7 +71,7 @@ def test_awq_loading(device, qdtype):
     
     model_save_path = "awq_model.pth"
     torch.save(m, model_save_path)
-    loaded_model = torch.load(model_save_path)
+    loaded_model = torch.load(model_save_path, assign=True)
```

Comment:
assign=True is an arg for `model.load_state_dict(...)` I think, see https://pytorch.org/ao/stable/serialization.html